Dockerfile

```diff
@@ -87,14 +87,13 @@ ENV HOME /home/app
 #####
 
 # TODO: use packrat (or something else) for R package management
-COPY packages.R $HOME
-RUN Rscript packages.R
-
-# install custom packages from R/pkgs/**
-COPY local_install.R $HOME
-COPY R/pkgs $HOME/R/pkgs
-RUN Rscript local_install.R
-
+RUN Rscript -e "install.packages('packrat',repos='https://cloud.r-project.org/')"
```

Comment:
set packrat (and arrow) versions? otherwise this will always pull latest

Reply:
packrat is deprecated, so any updates are bugfixes.  this was the only way i could get arrow to install, will think about a better solution.  (this is what arrow::install_arrow runs)